index.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Task2</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="container">
+        <div class="card-container">
+            <div class="text-container">
+                <button>
+                    <label for="button">Contact Us</label>
+                </button>
+                <div class="text">
+                    <h3>
+                        Reach out to us
+                    </h3>
+                    <p>
+                        Our mission is to revolutionize the resume writing process with AI technology and make it
+                        accessible to everyone.
+                    </p>
+                </div>
+            </div>
+            <div class="input-container">
```

Comment:
remove fixed width, layout should be flexible to desktop screens 900px -> any size

Reply:
Also fix the width of input container to maximum of e.g 1000px and align it to center